astro_2.0/features/ViewProposal/components/ProposalComments/components/Comment/Comment.tsx

```diff
@@ -141,17 +165,19 @@ export const Comment: FC<CommentProps> = ({
               tooltip="Remove"
             />
           </div>
+        ) : (
+          <div className={styles.commentControlButtonWrapper}>
+            <ActionButton
+              className={styles.commentControlButton}
+              tooltipPlacement="top"
+              iconClassName={styles.commentControlIcon}
+              iconName="commentBlock"
+              onClick={handleReportAction}
+              tooltip="Report"
+            />
+          </div>
```

Comment:
Is there sense to move it to render method, like
```
function renderActionButton(icon: IconName, action: () => void, tooltip) {
  return (
    <div className={styles.commentControlButtonWrapper}>
            <ActionButton
              className={styles.commentControlButton}
              tooltipPlacement="top"
              iconClassName={styles.commentControlIcon}
              iconName={icon}
              onClick={action}
              tooltip={tooltip}
            />
          </div>
  )
}
```

and reuse it in line 158 and 169?

Reply:
done!